### PR TITLE
update master sizes

### PR DESCRIPTION
**Pull request by rasheedamir**
change minimum and recommended masters sizes

### CODE REVIEW

File: docs/content/requirements/requirements.md [hunk 1]
```diff
@@ -29,25 +29,25 @@ The overall minimum resource requirements are:
...
-| Pipeline | 4 x 16 x 100 | 1 | 4 | 16 | 100 |
-| Worker | 4 x 16 x 100 | 3 | 12 | 48 | 300 |
-| **Total** | | **11** | **44** | **240** | **1100** |
+| Master | 6 x 24 x 120 | 3 | 12 | 72 | 300 |
+| Infra | 4 x 16 x 120 | 2 | 8 | 32 | 200 |
+| Monitoring | 4 x 32 x 120 | 1 | 4 | 32 | 100 |
+| Logging | 4 x 16 x 120 | 1 | 4 | 16 | 100 |
+| Pipeline | 4 x 16 x 120 | 1 | 4 | 16 | 100 |
+| Worker | 4 x 16 x 120 | 3 | 12 | 48 | 300 |
+| **Total** | | **11** | **50** | **216** | **1100** |
```
Comment (karl-johan-grahn) [inline docs/content/requirements/requirements.md]: don' think 50 will change, total is still 44

**Review by karl-johan-grahn — CHANGES_REQUESTED** [review]

File: docs/content/requirements/requirements.md (hunk 1 above)
Comment (rasheedamir) [reply docs/content/requirements/requirements.md]: it will i changed from 4 vcpus to 6 vcpus for master

**Comment by rasheedamir** [comment]
i have to update the storage! let me fix that as well

File: docs/content/requirements/requirements.md (hunk 1 above)
Comment (karl-johan-grahn) [reply docs/content/requirements/requirements.md]: you need to update row 44 column 4 then from 12 to 18, which will total up to 48, not 50 on row 50

**Comment by karl-johan-grahn** [comment]
@rasheedamir can you please update this soon

File: docs/content/requirements/requirements.md (hunk 1 above)
Comment (karl-johan-grahn) [reply docs/content/requirements/requirements.md]: I fixed it

**Review by karl-johan-grahn — APPROVED** [review]

**Pull request merged**